xla/service/gpu/cublas_cudnn.cc

```diff
@@ -124,6 +127,22 @@ bool IsCustomCallToDnnNorm(const HloInstruction& hlo) {
   return target == kCudnnNormCallTarget;
 }
 
+bool IsFwdCustomCallTofMHAF8(const HloInstruction& hlo) {
+  if (hlo.opcode() != HloOpcode::kCustomCall) {
```

Comment:
nit: here and below, we could maybe synthesize the function into a one-liner: `return hlo.opcode() == HloOpcode::kCudnnfMHASoftmaxF8CallTarget && hlo.custom_call_target() == kCudnnfMHASoftmaxF8CallTarget;`